docs/api/Rectangle.md

```diff
@@ -47,6 +47,25 @@ Adjust the rectangle by scaling it. The `scaleHeight` argument can be ommited to
 
 Return this rectangle (useful if you want to chain the calls).
 
+## Change the coordinates basis
```

Comment:
as you asked for general API comments, `basis` is a new concept that people might be unfamiliar with. It wasn't immediately clear, but reading it obviously makes sense. Perhaps people are more used to the idea of `coordinate "space"`? or yeah `origin`? I guess there are a few generic Maths-y terms. But I don't have any real objection to basis, just a passing comment.

Reply:
Interesting. The "proper" maths-y term is `basis` tho, which means "system of coordinates" (in some specific cases that happen to be the normal case 😄) :)

But maybe I can explain a bit more what is a `basis` with some links to learn more

Reply:
hehe I guess emphasis on the "generic". I either can't remember or never studied linear algebra that deeply despite doing a CS degree. But yeah `changeCoordinateSystem`, `changeCoordinateOrigin` `changeCoordinateSpace` probably aren't that snappy as a method name, despite maybe being more math-noob friendly. Anyway, was just my first reaction :)